tsconfig.test.json

```diff
@@ -2,7 +2,10 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "declaration": false,
+    "inlineSources": false,
+    "rootDir": "",
+    "outDir": "",
```

Comment:
Do we output the test `index.js` to the repo root?

Reply:
Nope! Seemingly this "unsets" the `outDir` set in the base config. So it defaults to outputting it alongside the source, in the `test` directory.